project_management/tps-funding-application-20190429.md

```diff
@@ -0,0 +1,205 @@
+
+Metrics of success over 18 months:
+
+* 20 new chapters.
+* 100 authors (significant contributions to the books).
+* 200 contributors (any contribution to the books).
+* 1000 mailing list subscribers (currently 200).
+* 50 first pull requests.
+* 20 new contributors to other open source projects.
+
+
+### Route to impact
+
+*Max 300 words*
+
+We are committed to giving all early career data scientists the support they need to confidently publish reproducible, accurate, interpretable, and responsible research.
+We will support financial analysts, civil servants, people in third sector organisations, and domain experts who need to evaluate applications of data science and AI.
+_The Turing Way_ books will be accessible to everyone who wants to learn these skills and the content will adapt to accommodate the needs of these communities.
+
+Pathways:
+  * **Training Steering Group** - use of The Turing Way chapters in doctoral training.
+  * **Book dash events** - respond to community requests for content and build confidence in early career researchers to share their skills.
+  * **Research engineering and partnerships teams** - chapter on effectively scoping an interdisciplinary project.
+
+_The Turing Way_ will enable world class research by facilitating communication interdisciplinary teams.
+The chapters will cover the "bridging skills" needed for historians to collaborate with software engineers, clinicians with deep learning algorithm developers, and civil servants to develop ethical and responsible applications of AI technology.
+
+Pathways:
+  * **Living with machines** - bidirectional community support to build a useful resource for interdisciplinary teams.
+  * **Health programme** - chapters on responsible data sharing, analyses within a safe haven environment, and citizen science.
+  * **Policy programme** - chapters on ethical frameworks and bias in machine learning.
+
+_The Turing Way_ will promote best practice in data science at a global scale.
+It is a "meta-project" developed and documented openly from the start.
+We will nurture a community of contributors who will themselves support a healthy and sustainable open source ecosystem.
+
+Pathways:
+  * **Jupyter and Microsoft Azure development communities** -  contribute and promote _The Turing Way_ across their networks of influence, and to listen to "pain points" in their tools experienced by community members.
+  * **Carpentries community** - co-design training materials and ensure _The Turing Way_ meets the highest pedagogical standards.
+
+### Research challenges
+
+*Max 300 words*
+
+Translating knowledge from one domain to another is always harder than it first appears.
+Tackling the *known unknowns* are the core goal of _The Turing Way_: providing a space for research teams to acquire the skills that they know they lack.
```

Comment:
Tackling the known unknowns *is* the core goal of The Turing Way

Reply:
✨ 🙊